arangod/V8Server/v8-voccursor.cpp

```diff
@@ -115,6 +110,9 @@ static void JS_CreateCursor(v8::FunctionCallbackInfo<v8::Value> const& args) {
 
     auto result = TRI_V8UInt64String<TRI_voc_tick_t>(isolate, id);
     TRI_V8_RETURN(result);
+  } catch (arangodb::basics::Exception const& exc) {
+    // Probably a coordinator soft shutdown:
+    TRI_V8_THROW_EXCEPTION_FULL(exc.code(), exc.message());
   } catch (...) {
     TRI_V8_THROW_EXCEPTION_MEMORY();
```

Comment:
I think we can get rid of the entire catch blocks sequence here, because the function ends with a `TRI_V8_TRY_CATCH_END`, which will expand to an exception handler anyway.

Reply:
OK. Good.

Reply:
I think here this does not work, since otherwise the `catch (...)` catches it and reports out of memory. I am inclined to leave this catch block in and only remove the one further down in the same file.